nuxt.config.ts

```diff
@@ -64,6 +71,11 @@ export default defineNuxtConfig({
 
   runtimeConfig: {
     public: {
+      scripts: {
+        crisp: {
+          id: ''
```

Comment:
empty string?

Reply:
use `mock`?
https://scripts.nuxt.com/docs/guides/registry-scripts#disabling-in-development

Reply:
it probably is not about disabling in dev, but env only works if id is `''`

Reply:
e.g. in doc


```
runtimeConfig: {
    public: {
      scripts: {
        cloudflareWebAnalytics: {
          // provide empty string so .env works
          token: '', // NUXT_PUBLIC_SCRIPTS_CLOUDFLARE_WEB_ANALYTICS_TOKEN
        },
      },
    },
  },
```